go.mod

```diff
@@ -50,6 +56,7 @@ require (
 	github.com/kevinburke/twilio-go v0.0.0-20200203063821-378e630e02da
 	github.com/logzio/logzio-go v0.0.0-20200316143903-ac8fc0e2910e
 	github.com/nukosuke/go-zendesk v0.10.1
+	github.com/onsi/ginkgo v1.16.5
```

Comment:
It looks like we missed `/v2` some in some import and Go automatically assumed we want to bring back Ginkgo v1.

Reply:
Yeah this was me fighting my IDE that insisted on removing the `_ github.com/onsi/ginkgo/v2` import I had originally. Should be resolved now.